app/views/notification_mailer/send_notification.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Hi <%= @user.full_name %>,</h1>
+<br/>
+<p><%= @message %></p>
+<br/>
+<p>Thanks,<br/>
+  your automatic e-mail system</p>
+<p>Irgendein Footer</p>
```

Comment:
Mixing english and german isn't that great. Also: This kind of shenenigans should not go into production code!

Reply:
👍🏻

Reply:
@Durborough Neither was it ever intended to. As stated before, I only got to work on it on Wednesday and intended and still intend to work on it on Monday / Tuesday. The creation of a pull request did come as a surprise to me.

Reply:
@gruendlich The PR was not opened to mark this work as done, but to have continuous/early feedback and include @Durborough .